lib/qs/payload_handler.rb

```diff
@@ -88,9 +88,9 @@ def log_complete(redis_item)
         'time'    => redis_item.time_taken,
         'handler' => redis_item.handler_class
       }
-      if (job = redis_item.job)
-        summary_line_args['job']    = job.name
-        summary_line_args['params'] = job.params
+      if (message = redis_item.message)
+        summary_line_args['name']   = message.route_id
```

Comment:
@kellyredding - The key is what is displayed in the summary line. So it will show `name="job|build_pdf"`. It can be `route-id` but I wasn't sure the basic user should be aware of route ids and see those in their logs.